Source/PercentageGradient.swift

```diff
@@ -7,18 +7,53 @@ public enum GradientDirection {
 
 }
 
+public struct GradientPoints {
+    
+    public var start: CGPoint
+    public var end: CGPoint
+    
+}
+
+public enum GradientAngle {
+    
+    case custom(start: CGPoint, end: CGPoint)
+    case defaultAngle
+    case bottomLeftToTopRight
+    case topLeftToBottomRight
+    case horizontal
+    case vertical
+    
+    func gradientPoints() -> GradientPoints {
+        switch self {
+        case .custom (let start, let end):
+            return GradientPoints(start: start, end: end)
+        case .bottomLeftToTopRight:
+            return GradientPoints(start: CGPoint(x: 0.0, y: 1.0), end: CGPoint(x: 1.0, y: 0.0))
+        case .defaultAngle:
```

Comment:
I think this would match general Apple conventions more as a `static let defaultAngle = GradientAngle.vertical` than a case that you can switch on, especially since it has the same as `.vertical`.